os/linux/weston/plugin/compositor-iahwc.c

```diff
@@ -1495,6 +1495,7 @@ static int vsync_callback(iahwc_callback_data_t data, iahwc_display_t display,
 
   if (output->pageflip_timer && output->frame_commited) {
     wl_event_source_timer_update(output->pageflip_timer, 1);
+    // FIXME: This fails to launch weston.
```

Comment:
I forgot to remove this code. This should never be there in the first place. Placing it here would race with the main thread and will set a wrong state in the state machine.